src/eos-dropbox-app

```diff
@@ -235,6 +243,25 @@ class DropboxLauncher():
             logging.info("Not the main launcher instance. Exiting")
             self._quit()
 
+    # This method is required for compatibility with xdg-desktop-portal < 0.6, for
+    # installations that might have updated the DropBox app before the base OS.
+    def _fallbackToOpenURIMethod(self, path):
+        handle = self._proxy.OpenURI('(ssa{sv})',
+                                     '',                          # Parent window ID
+                                     "file://{}".format(path),    # URI
+                                     None) # Options
```

Comment:
`# Options` should be aligned with the other comments

Reply:
It should indeed. Oops!